modyn/selector/internal/selector_manager.py

```diff
@@ -188,7 +188,7 @@ def uses_weights(self, pipeline_id: int) -> bool:
     def _instantiate_strategy(self, selection_strategy: dict, pipeline_id: int) -> AbstractSelectionStrategy:
         strategy_name = selection_strategy["name"]
         maximum_keys_in_memory = selection_strategy["maximum_keys_in_memory"]
-        config = selection_strategy["config"] if "config" in selection_strategy else {}
```

Comment:
I don't think this is a semantically equivalent change. 
<img width="601" alt="image" src="https://github.com/eth-easl/modyn/assets/100585941/948fce7b-a640-4de4-8927-ae87f4552072">

We should be more careful making changes in PRs.

Reply:
Ah I guess I get it. The inner class `config` seems to be flattened!